src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9/eventmottakoghistorikk/punsj/EventRepository.kt

```diff
@@ -0,0 +1,223 @@
+package no.nav.k9.los.nyoppgavestyring.domeneadaptere.k9.eventmottakoghistorikk.punsj
+
+import kotliquery.TransactionalSession
+import kotliquery.queryOf
+import kotliquery.sessionOf
+import kotliquery.using
+import no.nav.k9.los.aksjonspunktbehandling.objectMapper
+import no.nav.k9.los.domene.lager.oppgave.v2.TransactionalManager
+import no.nav.k9.los.nyoppgavestyring.feilhandtering.DuplikatDataException
+import no.nav.k9.los.nyoppgavestyring.feilhandtering.FinnerIkkeDataException
+import org.postgresql.util.PSQLException
+import java.util.*
+import javax.sql.DataSource
+
+
+class EventRepository(
+    private val dataSource: DataSource,
+    private val transactionalManager: TransactionalManager
+) {
+
+    fun lagre(event: PunsjEventV3Dto): EventLagret? {
+        val json = objectMapper().writeValueAsString(event)
+        return try {
+            using(sessionOf(dataSource)) {
+                it.run(
+                    queryOf(
+                        """
+                            insert into eventlager_punsj(ekstern_id, ekstern_versjon, eventnr_for_oppgave, "data", dirty) 
```

Comment:
Vi kunne kanskje ha kalt det `payload `eller `eventdata` for å slippe navnekollisjon?